<!--
  Launcher for the antobot_heading node written in Python
-->

<launch>
   <node pkg="antobot_heading" type="antobot_heading_node.py" name="antobot_heading_node_py" output="screen"> 
    <param name="new" type="bool" value="True" />
    <param name="gps_topic" type="string" value="/am_gps_urcu" />
  	<param name="imu_topic" type="string" value="/imu/data" />
  	<param name="odometry_topic" type="string" value="/odometry/filtered" />
    <param name="wheel_odometry_topic" type="string" value="/am_robot/odom" />
  </node>
</launch> 
